src/main/java/com/gamzabat/algohub/enums/EmailType.java

```diff
@@ -0,0 +1,15 @@
+package com.gamzabat.algohub.enums;
+
+import lombok.Getter;
+
+@Getter
+public enum EmailType {
+	RESET_PASSWORD("resetPassword"),
+	EMAIL_VALIDATION("emailValidation");
+
+	private final String value;
+
+	EmailType(String value) {
+		this.value = value;
+	}
+}
```

Comment:
`EmailTemplateStrings`랑 같이 보면서 든 생각인데요 아래처럼 이메일 타입 별로 필요한 정보를 enum에 모아두는건 어떻게 생각하시나요?

```suggestion
@Getter
@RequiredArgsConstructor
public enum EmailType {
	RESET_PASSWORD(
		"resetPassword",
		"[AlgoHub] 비밀번호 찾기",
		"비밀번호 재설정",
		"버튼을 누르면 새로운 비밀번호 설정을 위한 페이지로 이동합니다. <br/>3시간내 비밀번호를 재설정하지 않으면, 링크는 만료됩니다.",
		"비밀번호 재설정",
		"https://algohub.kr/reset-password"
	),
	EMAIL_VALIDATION(
		"emailValidation",
		"[AlgoHub] 이메일 유효성 검사",
		"메일 인증",
		"버튼을 누르면 알고헙 계정에 사용할 메일 인증이 완료됩니다. <br/>링크의 제한시간은 3분으로, 이후 링크는 만료됩니다.",
		"인증",
		"https://algohub.kr/sign-up"
	);

	private final String value;
	private final String subject;
	private final String title;
	private final String content;
	private final String button;
	private final String url;
}
```


필드 수가 많아질수록 가독성은 매우 구려진다는 단점이 있긴 합니다. (하나하나 매칭해서 뭐가 무슨 값을 의미하는지 찾아야 하니까..) 하지만 이메일 타입 별 정보를 모아서 관리하면 여기저기 코드 다녀보지 않고도 여기서 모든 정보를 알 수 있을 것 같다는 생각이 들어요. 

+) 위의 방식으로 변경되면 아래 `EmailService`에서 context를 만드는 로직도 굉장히 단순해집니다. (suggestion 달아둘게요)